deps/build.jl

```diff
@@ -1,10 +1,106 @@
-using Compat
-using Libdl 
+using BinaryProvider # requires BinaryProvider 0.3.0 or later
```

Comment:
Should this be added to the REQUIRE file?

Reply:
(it seems to work without it, but I assume that's because it's installed by some dependant library)

Reply:
Yes, it needs to be in REQUIRE.